write-fonts/src/tables/variations.rs

```diff
@@ -39,3 +270,77 @@ impl<'a> FromObjRef<Option<read_fonts::tables::variations::Tuple<'a>>> for Vec<F
             .unwrap_or_default()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn point_pack_words() {
+        let thing = PackedPointNumbers {
+            is_all: false,
+            numbers: vec![1002, 2002, 8408, 12228],
+        };
+
+        let runs = thing.iter_runs().collect::<Vec<_>>();
+        assert!(runs[0].are_words);
+        assert_eq!(runs[0].last_point, 0);
+        assert_eq!(runs[0].points, &[1002, 2002, 8408, 12228]);
+
+        let bytes = crate::dump_table(&thing).unwrap();
+        let (read, _) = read_fonts::tables::variations::PackedPointNumbers::split_off_front(
+            FontData::new(&bytes),
+        );
+        assert_eq!(thing.numbers, read.iter().collect::<Vec<_>>());
+    }
+
+    #[test]
+    fn smoke_test_point_packing() {
+        let thing = PackedPointNumbers {
+            is_all: false,
+            numbers: vec![5, 25, 225, 1002, 2002, 2008, 2228],
+        };
+
+        let runs = thing.iter_runs().collect::<Vec<_>>();
+        assert!(!runs[0].are_words);
+        assert_eq!(runs[0].last_point, 0);
+        assert_eq!(runs[0].points, &[5, 25, 225]);
+
+        assert!(runs[1].are_words);
+        assert_eq!(runs[1].last_point, 225);
+        assert_eq!(runs[1].points, &[1002, 2002]);
+
+        assert!(!runs[2].are_words);
+        assert_eq!(runs[2].last_point, 2002);
+        assert_eq!(runs[2].points, &[2008, 2228]);
+
+        assert_eq!(runs.len(), 3);
+        let bytes = crate::dump_table(&thing).unwrap();
+        let (read, _) = read_fonts::tables::variations::PackedPointNumbers::split_off_front(
+            FontData::new(&bytes),
+        );
+        assert_eq!(thing.numbers, read.iter().collect::<Vec<_>>());
+    }
+
+    // <https://learn.microsoft.com/en-us/typography/opentype/spec/otvarcommonformats#packed-deltas>
+    #[test]
+    fn packed_deltas_spec() {
+        static EXPECTED: &[u8] = &[
+            0x03, 0x0A, 0x97, 0x00, 0xC6, 0x87, 0x41, 0x10, 0x22, 0xFB, 0x34,
+        ];
+
+        let deltas = PackedDeltas::new(vec![10, -105, 0, -58, 0, 0, 0, 0, 0, 0, 0, 0, 4130, -1228]);
+        let runs = deltas.iter_runs().collect::<Vec<_>>();
+        assert_eq!(runs[0], PackedDeltaRun::OneByte(&[10, -105, 0, -58]));
+        assert_eq!(runs[1], PackedDeltaRun::Zeros(8));
+        assert_eq!(runs[2], PackedDeltaRun::TwoBytes(&[4130, -1228]));
+
+        let bytes = crate::dump_table(&deltas).unwrap();
+        assert_eq!(bytes, EXPECTED);
+        let read = read_fonts::tables::variations::PackedDeltas::new(FontData::new(&bytes));
+        let decoded = read.iter().collect::<Vec<_>>();
+        assert_eq!(deltas.deltas.len(), decoded.len());
+        assert_eq!(deltas.deltas, decoded);
+        assert_eq!(bytes, EXPECTED);
+    }
+}
```

Comment:
In general I beleive you are missing tests for no items and for more items than the max per run?